test/dlc_tests/sanity/test_security_check.py

```diff
@@ -35,6 +37,21 @@ def test_security(image):
         run(f"docker rm -f {container_name}", hide=True)
 
 
+def run_scan(ecr_client, image):
+    scan_status = None
+    start_time = time()
+    ecr_utils.start_ecr_image_scan(ecr_client, image)
+    while (time() - start_time) <= 600:
```

Comment:
is 600 timeout enough?

Reply:
It is more than enough, yes. The ECR Scan only takes about 5 minutes, and though it is mostly reliable, I have placed a timeout to prevent test/job timeouts as a general principle.